main.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/almighty/almighty-core/migration"
 	"github.com/almighty/almighty-core/models"
 	"github.com/almighty/almighty-core/remoteworkitem"
+	"github.com/almighty/almighty-core/search"
```

Comment:
There is not much value in this PR. As you can see below, the Search repository is used.

Let's close this PR. You can do it in your PR.